pkgs/development/libraries/agda/iowa-stdlib/default.nix

```diff
@@ -26,6 +26,8 @@ mkDerivation (rec {
     platforms = lib.platforms.unix;
     # broken since Agda 2.6.1
     broken = true;
+    # remove once unbroken
+    hydraPlatforms = lib.platforms.none;
```

Comment:
Would that be necessary for every broken package? If yes, it should be set by the agda framework I guess.

Reply:
It is not necessary, but it reduces the noise on Hydra quite a bit. For haskell we do something like you are suggesting via `hackage2nix`, you could probably do it in your builder, yeah.

Reply:
It may be. Since I've merged this in, I'll start a new evaluation on https://hydra.nixos.org/jobset/nixpkgs/haskell-updates.

Can you keep an eye on the `Evaluation Errors` tab over the next hour or so?  If there are additional evaluation errors for the agda-related packages, can you send PRs setting `hydraPlatforms` to `none`?

Reply:
Ok, but then let's set `hydraPlatforms = none` in the agda builder. I'll try to do that in an MR, and you can tell me whether it's working.

Reply:
See https://github.com/NixOS/nixpkgs/pull/131089